Mathlib/SetTheory/Surreal/Multiplication.lean

```diff
@@ -9,8 +9,8 @@ import Mathlib.SetTheory.Surreal.Basic
 /-!
 ### Surreal multiplication
 
-In this file, we show that multiplication of surreal numbers is well-defined, and thus the
-surreal numbers form a linear ordered commutative ring.
+In this file, we show that multiplication of surreal numbers is well-defined, and thus the surreal
```

Comment:
I believe this is a case of semantic line break, where the line break is not placed the furthest along the sentence as it can without breaking words, but rather the furthest along the sentence without breaking a grammatical block.